airbyte-commons-worker/src/test/java/io/airbyte/workers/WorkerUtilsTest.java

```diff
@@ -5,125 +5,22 @@
 package io.airbyte.workers;
 
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.mockito.Mockito.doAnswer;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoInteractions;
-import static org.mockito.Mockito.when;
 
 import com.fasterxml.jackson.databind.JsonNode;
-import io.airbyte.config.Configs;
-import io.airbyte.config.EnvConfigs;
 import io.airbyte.config.StandardSync;
 import io.airbyte.config.StandardSyncInput;
 import io.airbyte.protocol.models.AirbyteStreamNameNamespacePair;
-import io.airbyte.workers.internal.HeartbeatMonitor;
 import io.airbyte.workers.test_utils.TestConfigHelpers;
-import java.time.Duration;
-import java.time.temporal.ChronoUnit;
 import java.util.Map;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.function.BiConsumer;
 import org.apache.commons.lang3.tuple.ImmutablePair;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.DisplayName;
-import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 @SuppressWarnings("PMD.JUnitTestsShouldIncludeAssert")
 class WorkerUtilsTest {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(GentleCloseWithHeartbeat.class);
-
-  @Nested
-  class GentleCloseWithHeartbeat {
```

Comment:
i don't think this is testing anything that is still used. but @davinchia  this is where i feel least sure. it looks like in the original orchestrator implementation some of this code got copied over. but i think it's not used.

Reply:
your comment looks right to me. 

best way to confirm this is to make sure the acceptance tests pass. I've fixed the PMD error so we can run the full build.